tests/keras/layers/wrappers_test.py

```diff
@@ -86,6 +86,12 @@ def test_TimeDistributed():
     outer_model.compile(optimizer='rmsprop', loss='mse')
     outer_model.fit(np.random.random((10, 3, 2)), np.random.random((10, 3, 3)), epochs=1, batch_size=10)
 
+    # test layers that need learning_phase to be set
+    model = Sequential()
+    model.add(wrappers.TimeDistributed(core.Dropout(.5), input_shape=(3, 2)))
+    model.compile(optimizer='rmsprop', loss='mse')
+    model.predict(np.random.random((10, 3, 2)))
```

Comment:
I am thinking a good test would be something like below, but account for the cases where dropout eliminates the entire input so the sum would be zero

```python
x = np.array([
    [[1, 1], [2, 2], [-1, -1]]
])
y = model.predict(x)
np.testing.assert_allclose(x.sum(axis=2), y.sum(axis=2), atol=1e-05)
```

To get that to pass though I would need to sort out the mask issue. It's unclear to me if it's a model issue (my fault), or `TimeDistributed` doesn't yet handle masking correctly in all cases (https://github.com/fchollet/keras/issues/4786 and https://github.com/fchollet/keras/issues/5212)